en-US/Players_Guide/06_Spellcasting/Spell_Lists/Sorcerer_Spells.md

```diff
@@ -4,9 +4,10 @@
 ##### Cantrips (0 Level)
 
 - [Acid Splash](#Acid_Splash_acid_splash)
-- Chill Touch
-- Dancing Lights (concentration)
-- Fire Bolt
+- [Chill Touch](#Chill_Touch_chill_touch)
+- [Dancing Lights](#Dancing_Lights_dancing_lights) (concentration)
+- [Fire Bolt](#Fire_Bolt_fire_bolt)
+- [Friends](#Friends_friends)
```

Comment:
This seems to be a new A5e spell that was not yet in the spell list. According to the A5e spell document however, it is a Sorcerer spell, so I added it here.